pkgs/top-level/python-packages.nix

```diff
@@ -4196,6 +4196,28 @@ in modules // {
       description = "pytest plugin with mechanisms for caching across test runs";
     };
   };
+  
+  pytestdjango = buildPythonPackage rec {
+    name = "pytest-django-${version}";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pytest-django/${name}.tar.gz";
+      sha256 = "1mmc7zsz3dlhs6sx4sppkj1vgshabi362r1a8b8wpj1qfximpqcb";
+    };
+    
+    # doing this to allow depending packages to find
+    # pytest's binaries
+    pytest = self.pytest;
```

Comment:
Yes, that works from a _runtime_ perspective, but if a depending package is able to grab the specific pytest's outpath, it enables it to e.g. build a wrapper shell script which directly references the specific pytest's binary rather than having to rely on runtime `PATH`.